rosbag_snapshot/src/snapshotter.cpp

```diff
@@ -62,6 +62,11 @@ const ros::Duration SnapshotterTopicOptions::INHERIT_DURATION_LIMIT = ros::Durat
 const int32_t SnapshotterTopicOptions::INHERIT_MEMORY_LIMIT = 0;
 const int32_t SnapshotterTopicOptions::INHERIT_COUNT_LIMIT = 0;
 
+static bool is_topic_name_pattern(const std::string& s)
+{
+  return s.find_first_of(".*+?()[]{}\\") != std::string::npos;
```

Comment:
Please add to the README with information about the pattern matching syntax. This is most important so that it doesn't get mixed up with [glob syntax](https://en.wikipedia.org/wiki/Glob_(programming))